test/unit/gateways/authorize_net_test.rb

```diff
@@ -613,6 +613,25 @@ def test_supports_scrubbing?
     assert @gateway.supports_scrubbing?
   end
 
+  def test_successful_apple_pay_authorization_with_network_tokenization
+    credit_card = network_tokenization_credit_card('4242424242424242',
+      :payment_cryptogram => "111111111100cryptogram"
+    )
+
+    response = stub_comms do
+      @gateway.authorize(@amount, credit_card)
+    end.check_request do |endpoint, data, headers|
+      parse(data) do |doc|
+        assert_equal credit_card.payment_cryptogram, doc.at_xpath("//creditCard/cryptogram").content
+        assert_equal credit_card.number, doc.at_xpath("//creditCard/cardNumber").content
```

Comment:
This is this expected to be `nil`, yes? Can we maybe just explicitly check that it's blank?